unblob/processing.py

```diff
@@ -34,17 +34,19 @@
 DEFAULT_PROCESS_NUM = multiprocessing.cpu_count()
 
 
+@attr.define(kw_only=True)
+class ExtractionConfig:
```

Comment:
Since this is used from multiple places, I would put this class definition be in the `models.py`

Reply:
I was unsure about suggesting this as well. From one hand it would match better with the existing code structure, on the other hand I find this `models` module containing a bunch of unrelated datastructures a bit odd.

Reply:
Yeah, me too. I was just going with the consistency.... On the other hand, I have already approved this when I wrote this comment, so it can be merged without this modification :smile: That's my way of saying "just a small suggestion, but don't care that much".

Reply:
Thanks for the review, I was thinking about moving to models as well (consistency), I merge it as it is now, we can move to models in another PR if that is a better place - later.